azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -267,6 +267,66 @@ func resourceStorageAccount() *schema.Resource {
 								},
 							},
 						},
+
+						"change_feed": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"retention_in_days": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(1, 146000),
+									},
+								},
+							},
+						},
+
+						"versioning_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+
+						"default_service_version": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "2020-06-12",
+							ValidateFunc: validate.BlobPropertiesDefaultServiceVersion,
+						},
+
+						"last_access_time_tracking_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type:         schema.TypeString,
+										Optional:     true,
+										Default:      "AccessTimeTracking",
+										ValidateFunc: validation.StringInSlice([]string{"AccessTimeTracking"}, false),
+									},
+									"granularity_in_days": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										Default:      1,
+										ValidateFunc: validation.IntInSlice([]int{1}),
+									},
+									"blob_type": {
+										Type:     schema.TypeList,
+										Optional: true,
+										Computed: true,
+										Elem: &schema.Schema{
+											Type:         schema.TypeString,
+											ValidateFunc: validation.StringInSlice([]string{"blockBlob"}, false),
+										},
```

Comment:
a few things here:

1. `granularity_in_days` and `blob_type` can't be set, so should be computed-only
2. there's a single possible value for `name` and `blob_type` - so these likely shouldn't be exposed

as such this block could likely be removed in place of a single field for `last_access_time_tracking_policy_enabled` - unless there's plans by the service team to add other functionality here?

Reply:
Yes. The service plans to support the `granularity_in_days` and `blob_type` in the future. I just thought that it may be hard to covert `last_access_time_tracking_policy` from bool to list in the future.
Currently `azurerm_management_policy` is dependent on the `last_access_time_tracking_policy`.
But I will change it into bool now.